packages/core/src/select.js

```diff
@@ -11,21 +13,30 @@ type PluralForms = {
 
 type PluralProps = {
   value: number,
-  offset?: number
+  offset?: number,
+  locales?: string,
+  options?: {}
 } & PluralForms
 
+declare var Intl: {
+  NumberFormat: any
+}
+
 const _plural = type => (i18n: any) => ({
   value,
   offset = 0,
+  locales = i18n.locales || i18n.language,
+  options,
   other,
   ...pluralForms
 }: PluralProps): string => {
   const diff = value - offset
+  const diffAsString = number(locales, options)(diff)
   const translation =
     pluralForms[value.toString()] || // exact match
     pluralForms[i18n.pluralForm(diff, type)] || // plural form
     other // fallback
-  return translation.replace("#", diff.toString())
+  return translation.replace("#", diffAsString)
```

Comment:
@tricoder42 Shouldn't the value at: https://github.com/lingui/js-lingui/blob/0d0aa92e334909a27ec0a38b2c99559e3c4558e5/packages/core/src/context.js#L14 be also passed through the number() function?

Reply:
I'm not sure if it should be implicit. I see your point, thought.

One possible workaround might be using `number` formatter directly:
```{value, plural, one {# volt} other {# volts}```

vs

```{value, plural, one {{value, number, ...} volt} other {{value, number, ...} volts}}```

but that wouldn't work when you use `offset`.

If we want to use `number` formatter directly, we need to pass format options to plural as well.